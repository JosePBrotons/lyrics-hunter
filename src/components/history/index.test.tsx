import React from 'react';
import renderer from 'react-test-renderer';
import History from '.';

test('Renders history component correctly', () => {
    const props = {
        navigation: null
    };
    const historyCmp = renderer.create(<History {...props} />).toJSON();
    expect(historyCmp).toMatchSnapshot();
});