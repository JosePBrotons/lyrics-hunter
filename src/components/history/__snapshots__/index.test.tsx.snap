// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders history component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(233, 233, 233, 1)",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingHorizontal": 16,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/img/lyrics.png",
      }
    }
    style={
      Object {
        "height": 200,
        "marginBottom": 15,
        "width": 200,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "rgba(69, 69, 69, 1)",
        "fontFamily": "Nunito-Bold",
        "fontSize": 22,
        "lineHeight": 30,
        "textAlign": "center",
      }
    }
  >
    [missing "en.history.emptyState" translation]
  </Text>
</View>
`;
